src/webhook_launcher/settings.py

```diff
@@ -120,6 +120,13 @@
     from django_auth_ldap.config import LDAPSearch
     import logging
 
+    AUTH_LDAP_BIND_DN = ""
+    if config.has_option('ldap', 'ldap_bind_dn'):
+        AUTH_LDAP_BIND_DN = config.get('ldap', 'ldap_bind_dn', raw=True)
```

Comment:
Unfortunately 793673ed58b8d5da5379f1a3a71e0af3c84bafc9 doesn't explain it either, some ldap entries use raw=True, others don't.  Local consistency then, ok.